python/run-tests.py

```diff
@@ -109,40 +121,35 @@ def run_pypi_packaging_tests(root_dir):
     # uninstall packages if they exist
     run_cmd(["pip3", "uninstall", "--yes", "delta-spark", "pyspark"], stream_output=True)
 
-    print("### Clearing Delta artifacts from ivy2 and mvn cache")
-    run_cmd(["rm", "-rf", "~/.ivy2/cache/io.delta/"], stream_output=True)
-    run_cmd(["rm", "-rf", "~/.m2/repository/io/delta/"], stream_output=True)
+    # No need to rm -rf local .ivy and .m2 io/delta caches.
+    # They were deleted in `prepare`, and then new artifacts were locally published to them
```

Comment:
this comment does not make sense in the context of the current code. its like saying "i did not steal a bank" out of the blue in a conversation, it raises more questions and confuses people. :)

Reply:
noted. good point.